web/themes/joinup/templates/block/block--local-actions-block.html.twig

```diff
@@ -7,6 +7,6 @@
 #}
 {% block content %}
   {% if content %}
-    <nav class="tabs tabs--vertical mdl-tabs__tab-bar">{{ content }}</nav>
```

Comment:
There's a question in the ticket: `Fixing this in the theme will be simple, but maybe we will need to do some digging to find the exact reason why this mistake was initially made.`
You might want to reply to that it seems there's no specific business reasons: this was introduced quite early in general styling of elements: https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-2565
It seems a safe fix to me.